ember-app/app/initializers/sockets.js

```diff
@@ -36,6 +37,7 @@ export function initialize(container, application){
         return callback;
       },
       unsubscribe: function(channel, callback) {
+        if(channel == "null") { return Ember.K; }
```

Comment:
I figured it was something weird like that, but wanted to clarify. Maybe switch to `===`?